js/load-scene.js

```diff
@@ -97,15 +97,19 @@ AFRAME.registerComponent('loadscene', {
       browserReposition();
     }
 
+    function onStartClick() {
+      introAudioEl.pause();
+      // container is only visible once get-started button is clicked.
+      containerEl.setAttribute('visible', true);
+      // remove button from targettable raycaster objects.
+      getStartedButton.removeAttribute('data-clickable');
```

Comment:
this line and the line below mean that when I enter VR, press the **Get Started** button, exit VR, and re-enter VR, the **Get Started** modal does not appear ever again. that seems like fine and expected behaviour to me. but wanted to check if that confirms with the behaviour that y'all want.

Reply:
I would make everything re-settable so that you can restart, but this is how it works elsewhere too https://github.com/EFForg/spot_the_surveillance/blob/master/js/clickonce.js#L8

Reply:
Also should mention, when you "restart" using the option in the EFF menu, it reloads the page. 

I guess if you look at VR as a _state_, then it should not restart when you exit VR.  This is standard A-Frame behavior really.